### ISSUE TITLE
Error when closing window splits

### ISSUE THREAD

**Issue by Xemptuous** [gorbit99/codewindow.nvim#25]
I have two problems, but mostly this one: 

NeoVim: 0.7.2

When having multiple splits open - as well as the codewindow - if I switch between the two splits and then close either one, I get an error message like below, but with different window ID's:
```Error detected while processing WinScrolled Autocommands for "<buffer=1>":
Error executing lua callback: ...e/nvim/plugged/codewindow.nvim/lua/codewindow/window.lua:72: Invalid window id: 1009
stack traceback:
	[C]: in function 'nvim_win_get_width'
	...e/nvim/plugged/codewindow.nvim/lua/codewindow/window.lua:72: in function 'get_window_config'
	...e/nvim/plugged/codewindow.nvim/lua/codewindow/window.lua:87: in function <...e/nvim/plugged/codewindow.nvim/lua/codewindow/window.lua:84>
```
If I open the split and close it immediately with `<C-w><q>`, then all is fine, but it is the switching that causes this issue.
I have tried disabling various other plugins to try and pin-point the issue to no avail.

My Plugged init:
```
Plug 'gorbit99/codewindow.nvim'
[...]
require("codewindow").setup()
require('codewindow').apply_default_keybinds()
```
I also have no tree-sitter colors showing up, and not sure if it's tied to this issue.

**Comment by Xemptuous** [comment]
Fixed with update to 0.8.0

**Comment by gorbit99** [comment]
I don't support nvim versions < 0.8, because it's a lot of code (and can affect minimap performance).

If someone really needs it to work in 0.7.2, they need to redefine and link the new treesitter highlight groups to the old ones (@string.literal -> TSStringLiteral)

The other issue I think I pushed a fix to last night.
